### ISSUE TITLE
Maze Game 

### ISSUE THREAD

**Issue by chandu6111** [geekymeeky/JS-games#60]
**Describe the solution you'd like**
a Maze game where you will be having 4 levels at which you can play and try to get to your goal at last, when you will be reaching your goal(cabbage) by moving your player(bug) with keyboard, it will display the number of total steps in which you have completed the task.

- [x] DCP21

**Comment by chandu6111** [comment]
Hey @kanishkaa24 or @geekymeeky could you assign me this issue?

**Comment by kanishkaa24** [comment]
> Hey @kanishkaa24 or @geekymeeky could you assign me this issue?

Go ahead

**Comment by geekymeeky** [comment]
@chandu6111 any update?

**Comment by chandu6111** [comment]
Hey @geekymeeky I am presently stuck with exams and all, please give me some time, I will definitely submit it in a week. Sorry for the inconvenience!

**Comment by geekymeeky** [comment]
@chandu6111  I understand your situation, but if you haven't begun working on it yet, I'll assign issue #115 to the creator, and you can work on other issues after your examinations.

**Comment by chandu6111** [comment]
> @chandu6111 I understand your situation, but if you haven't begun working on it yet, I'll assign issue #115 to the creator, and you can work on other issues after your examinations.

No, @geekymeeky I have begun the work already, but I stopped it at some point. It was 70% done already. I will make a PR in few days(3-4)

**Comment by aditya687** [comment]
I want to work on this issue @geekymeeky
